src/Applicator/ApplicatorInterface.php

```diff
@@ -20,4 +20,17 @@ public function supports(Filterable $filterable): bool;
     public function applyTo(FilterInterface $filter, Filterable $filterable): Filterable;
 
     public function setEscape(EscapeInterface $escape): void;
+
+    /**
+     * Prepared values for applied filter
+     *
+     * @example
+     * $filterWithOperator = new FilterWithOperator('title', 'foo', '=', 'eq');
+     * $preparedValues = $simpleSqlApplicator->getPreparedValue($filter);  // ['title_eq' => 'foo']
+     *
+     * @example
+     * $filter = new FilterIn('id', [1, 2]);
```

Comment:
This annotation uses an example with `FilterIn` which is not implemented yet, but since it is just an example I don't see it as a problem.